js/uganda/ug_onEachFeature_fns.js

```diff
@@ -28,7 +28,8 @@ function OEF(layer, type) {
             popup_lines.push(`<strong>Number of Cases:</strong> ${cases}`)
         }
     }
-    if (type === "Cases per District") {
+    if (type === "Cases per District", "Border Districts", "Cases per 100,000 people", "Poverty Percentage", "Population Density", "Elderly Percentage(Over 60 in age)", 
```

Comment:
This change is not that clear for me. Please explain what it is that you're trying to achieve here.

Reply:
Am trying to make the number of cases a constant thing in all the popups for all layers so you don't need to keep going back to the cases layer just to see how many cases are in a district via the popup

Reply:
The solution to this is to add the cases to all the layers and remove this if statement.
 See if you can achieve that, @lynnnakiganda